docs/get-started/frameworks/react-native-web-vite.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Storybook for React Native Web
+sidebar:
+  order: 5
+  title: React Native Web
+---
+
+Storybook for React Native Web is a [framework](../../contribute/framework.mdx) that makes it easy to develop and test UI components in isolation for [React Native](https://reactnative.dev/) applications. It uses [Vite](https://vitejs.dev/) to build your components for web browsers. The framework includes:
+
+* ⚛️ React Native components
+* 📱 Mobile emulation in the browser
```

Comment:
mobile emulation is not being used so this would be misleading

Reply:
I mean that emulation seems like we're saying a phone runs in the browser

Reply:
removed!